### PR TITLE
Changes "/bin/bash" to "/usr/bin/env bash" to allow usage in modern macOS

### PR DESCRIPTION
**Pull request by Bradshaw**
The version of bash shipped with macOS is still out-dated (3.2.57 as of this pull request)

macOS has also added new security features preventing users from modifying the contents of /bin

This makes it impossible to symlink /bin/bash to an up to date version installed via other methods, such as homebrew  
   
Using a shebang like `#!/usr/bin/env bash` fixes this issue by letting the user's custom environment pick which bash executable to use

### CODE REVIEW

File: configure.sh [hunk 1]
```diff
@@ -167,12 +167,14 @@ GitHub repo: https://github.com/posva/configure-script"
 # Check if the file is unix or dos and convert it if
 # the used asked for it with -a
 function is_unix_valid() {
-  if grep -q "" $1 ; then
+  if grep -q "
```
Comment (posva) [inline configure.sh]: can you undo this change please

**Review by posva — CHANGES_REQUESTED** [review]
Thanks!

File: configure.sh (hunk 1 above)
Comment (Bradshaw) [reply configure.sh]: Oops, I did not notice that

**Review by Bradshaw — COMMENTED** [review]
Hmmm, apparently nano and also the text editor both drop the "^M" symbols and strip them before saving   
I edited only the first line and pushed a new version with only the first lines modified

**Review by posva — APPROVED** [review]
Thanks!
